src/Site_WP_Command.php

```diff
@@ -242,6 +242,17 @@ public function create( $args, $assoc_args ) {
 		EE\Utils\delem_log( 'site create end' );
 	}
 
+	/**
+	 * Creates database user for a site
+	 *
+	 * @param string $site_url URL of site
+	 *
+	 * @return string Generated db user
+	 */
+	private function create_site_db_user( string $site_url ) : string {
+		return substr( $site_url, 0, 73 ) . '-' . \EE\Utils\random_password( 6 );
```

Comment:
do not directly do substr first please check if the string has that much char.  And let's not keep limit of 60.
```
if ( strlen( $site_url ) > 53 ) {
      $site_url = substr( $site_url, 0, 53 );
}
return $site_url . '-' . \EE\Utils\random_password( 6 );
```

Reply:
Done